mxcubecore/HardwareObjects/GenericDiffractometer.py

```diff
@@ -831,7 +831,7 @@ def cancel_centring_method(self, reject=False):
             self.reject_centring()
 
     def start_manual_centring(self, sample_info=None, wait_result=None):
-        self.emit_progress_message("Manual 3 click centring...")
+        self.emit_progress_message("%s centring..." % self.CENTRING_METHOD_MANUAL)
```

Comment:
Use f-string?

Something like this maybe:

```suggestion
        self.emit_progress_message(f"{self.CENTRING_METHOD_MANUAL} centring...")
```